web/views/manifestations.py

```diff
@@ -36,11 +36,15 @@ def post(self):
             copyright_holder=copyright_holder,
             work_data=work)
 
-        res = {
-            'manifestation': manifestation.to_jsonld(),
-            'work': work.to_jsonld(),
-            'copyright': copyright.to_jsonld()
-        }
+        # Add the appropraite @id to the JSON-LD
+        res = {}
+        for (entity, id_template, key) in [
+                (copyright, '../right/{}', 'copyright'),
+                (manifestation, '{}', 'manifestation'),
+                (work, '../work/{}', 'work')]:
+            ld_data = entity.to_jsonld()
+            ld_data['@id'] = id_template.format(entity.persist_id)
+            res[key] = ld_data
```

Comment:
This is somewhat messy, since we have to use the relative URL here if we want to be JSON-LD compliant (related #14). It's not as nice as having just the ID, but stripping the path to get the ID is easy enough and I prefer this over a few alternatives on where to put the ID.